tests/test_analysis/test_activation_count.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+# Modified from
+# https://github.com/facebookresearch/fvcore/blob/main/tests/test_activation_count.py
+# pyre-ignore-all-errors[2]
+import typing
+import unittest
+from collections import Counter, defaultdict
+from typing import Any, Dict, List, Tuple
+
+import torch
+import torch.nn as nn
+from numpy import prod
+
+from mmengine.analysis import ActivationAnalyzer, activation_count
+from mmengine.analysis.jit_handles import Handle
+
```

Comment:
Do we need to test other operations?

Reply:
einsum share the same activation_count function with linear.